docs/src/index.ejs

```diff
@@ -49,8 +49,8 @@
   <script src="https://cdnjs.cloudflare.com/ajax/libs/react-dom/<%= htmlWebpackPlugin.options.versions.reactDOM %>/umd/react-dom<%= __PROD__ ? '.production.min' : '.development' %>.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/react-dom/<%= htmlWebpackPlugin.options.versions.reactDOM %>/umd/react-dom-server.browser<%= __PROD__ ? '.production.min' : '.development' %>.js"></script>
 
-  <script src="https://cdn.jsdelivr.net/npm/@babel/standalone@7/babel.js"></script>
-  <script src="https://cdn.jsdelivr.net/npm/@babel/preset-env-standalone@7/babel-preset-env.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/@babel/standalone@7.3/babel.min.js"></script>
```

Comment:
shouldn't we use the same version restriction as we have in our `devDependencies` - specifically, `^7.1`? Or this one is not working for the env where you are able to repro the problem (asking because for me it has worked)?

Reply:
The best way is to use `versions` from `package.json`, but it means that will also need to add `@babel/preset-env-standalone` to deps.

---

In fact, we not using `@babel/standalone` anywhere, so may be we can remove it from dependencies at all 🤔  I'm also not that we need `preset-env` there.

My proposal:
- remove the usage `@babel/preset-env`
- use `@babel/standalone` version from dependencies

Reply:
awesome!